### ISSUE TITLE
Can't compile v from README instructions

### ISSUE THREAD

**Issue by mgattozzi** [vlang/v#331]
Following these instructions: https://github.com/vlang/v#linux-and-macos

I tried to compile the compiler with the c code. While it mentions I can use "any directory" that doesn't seem to be the case due to bad paths seen below:

```
michael@eva-03 ~/v/compiler (git)-[master] % ./vc -o v .
errno=2 err='No such file or directory'
V panic: /Users/michael//code/v//os doesnt exist
```

**Comment by coolreader18** [comment]
You have to place `v` in the directory `~/code`, and if it doesn't exist you have to make it.

**Comment by bitsnaps** [comment]
> You have to place `v` in the directory `~/code`, and if it doesn't exist you have to make it.
that's what I did and I got another issue: `Segmentation fault: 11` <a href="https://github.com/vlang/v/issues/333">issue#333</a>

**Comment by Bleser** [comment]
`clang v.c -o vc`

**Comment by bitsnaps** [comment]
Fixed #issuecomment-504690571
